scripts/generate-genesis.py

```diff
@@ -0,0 +1,284 @@
+  supersim_output_deploy_configs_dir = os.path.join(supersim_output_dir, "deploy-configs")
+  supersim_output_l1_allocs_dir = os.path.join(supersim_output_dir, "l1-allocs")
+  supersim_output_addresses_dir = os.path.join(supersim_output_dir, "addresses")
+  supersim_output_l2_allocs_dir = os.path.join(supersim_output_dir, "l2-allocs")
+  supersim_output_l2_genesis_dir = os.path.join(supersim_output_dir, "l2-genesis")
+  supersim_output_rollup_dir = os.path.join(supersim_output_dir, "rollup-configs")
+
+  os.makedirs(supersim_output_dir, exist_ok=True)
+  os.makedirs(supersim_output_deploy_configs_dir, exist_ok=True)
+  os.makedirs(supersim_output_l1_allocs_dir, exist_ok=True)
+  os.makedirs(supersim_output_addresses_dir, exist_ok=True)
+  os.makedirs(supersim_output_l2_allocs_dir, exist_ok=True)
+  os.makedirs(supersim_output_l2_genesis_dir, exist_ok=True)
+  os.makedirs(supersim_output_rollup_dir, exist_ok=True)
+
+  monorepo_dir = os.path.abspath(monorepo_base_path)
+  monorepo_op_node_dir = os.path.join(monorepo_dir, 'op-node')
+  monorepo_contracts_bedrock_dir = os.path.join(monorepo_dir, 'packages', 'contracts-bedrock')
+  monorepo_deployment_dir = os.path.join(monorepo_contracts_bedrock_dir, 'deployments')
+  monorepo_deploy_config_dir = os.path.join(monorepo_contracts_bedrock_dir, 'deploy-config')
+  monorepo_forge_l1_dump_path = os.path.join(monorepo_contracts_bedrock_dir, 'state-dump-900.json')
+  monorepo_default_deploy_config_template_path = os.path.join(monorepo_deploy_config_dir, 'devnetL1-template.json')
+
+  paths = Bunch(
+    supersim_dir=supersim_dir,
+    supersim_contracts_dir=supersim_contracts_dir,
+
+    monorepo_dir=monorepo_dir,
+    monorepo_op_node_dir=monorepo_op_node_dir,
+    monorepo_contracts_bedrock_dir=monorepo_contracts_bedrock_dir,
+    monorepo_deployment_dir=monorepo_deployment_dir,
+    monorepo_deploy_config_dir=monorepo_deploy_config_dir,
+    monorepo_forge_l1_dump_path=monorepo_forge_l1_dump_path,
+    monorepo_default_deploy_config_template_path=monorepo_default_deploy_config_template_path,
+  )
+
+  l1_chain_id = 900
+
+  def get_l2_chain(i): 
+    return L2(l1_chain_id, 901 + i, deployer_addresses[i], supersim_dir)
+
+  l1_chain = L1(l1_chain_id, supersim_dir)
+  l2_chains = list(map(get_l2_chain, range(10)))
+
+  generate_deploy_configs(paths, l2_chains)
+  generate_l1_allocs_and_addresses(paths, l2_chains)
+  generate_combined_l1_allocs(paths, l1_chain, l2_chains)
+  generate_l2_allocs(paths, l2_chains)
+  generate_l1_genesis(paths, l1_chain, l2_chains)
+  generate_l2_genesis(paths, l1_chain, l2_chains)
+  
+def generate_deploy_configs(paths, l2_chains: list[L2]):
+  for l2_chain in l2_chains:
+    # Create deploy configs that will be used by the foundry deploy script
+    deploy_config = read_json(paths.monorepo_default_deploy_config_template_path)
+    deploy_config['l1ChainID'] = l2_chain.l1_chain_id
+    deploy_config['l2ChainID'] = l2_chain.l2_chain_id
```

Comment:
will the chain ids we setup here conflict with whats happening in our go code https://github.com/ethereum-optimism/supersim/blob/6073ce78ff834b4a400691b674bb16de5139a277/config/chain.go#L22-L40 or i'm assuming this is something we'll figure out once to start to consume these files

Reply:
yep! the next PR will use the generated files as the source of truth